discretisedfield/mesh.py

```diff
@@ -909,6 +900,15 @@ def plane(self, *args, n=None, **kwargs):
         return plane_mesh
 
     def __or__(self, other):
+        """Depricated method to check if meshes are aligned: use ``is_aligned``"""
```

Comment:
```suggestion
        #Deprecated method to check if meshes are aligned: use ``is_aligned``
```
I would remove the docstring to remove the operator from the documentation.